ui-guidelines/css/guidelines.css

```diff
@@ -165,8 +165,60 @@ pre {
 	font-size: 1em;
 	font-weight: bold;
 }
+
+/*==== TYPOGRAPHY ====*/
+
+.font-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	display: block;
+	margin-right: 5%;
+	border: 1px solid #fce3da;
+	border-radius: 8px;
+}
+.big-letter{
+	font-size: 6em;
+}
+.light {font-weight: 300;}
+.normal {font-weight: 400;}
+.medium {font-weight: 500;}
+.bold {font-weight: 700;}
+.text-size {margin-left: 1.5em;}
+
+h1 .text-h1 {font-size: 2em;}
+h2 .text-h2 {font-size: 1.5em;}
+h3 .text-h3 {font-size: 1.25em;}
```

Comment:
I'm not a great fan of this kind of notation. Maybe, it is a better idea to write it with the following convention: `h1 .text{} h2 .text {}`, isn't it? Anyway, from the HTML markup I see that the selector is incorrect. It should be: `h2.text-h2`, etc.